packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -811,7 +811,7 @@ class _CupertinoModalPopupRoute<T> extends PopupRoute<T> {
   bool get barrierDismissible => true;
 
   @override
-  bool get semanticsDismissible => false;
+  bool get semanticsDismissible => true;
```

Comment:
Is it correct to always return true, though? The documentation on `showCupertinoModalPopup` says its typically used to show an ActionSheet: https://developer.apple.com/design/human-interface-guidelines/ios/views/action-sheets/

And in a native action sheet on iOS the barrier is not a dismissible for VOiceOver users.

Reply:
that's a good point, perhaps it should be configurable? The issue I'm trying to solve is for the date picker demo, you can't dismiss the picker with semantics.

Reply:
I'm going to make this a parameter to allow it to be configured.